packages/instant/src/containers/selected_erc20_asset_amount_input.ts

```diff
@@ -69,7 +69,7 @@ const mapStateToProps = (state: State, _ownProps: SelectedERC20AssetAmountInputP
 
 const debouncedUpdateBuyQuoteAsync = _.debounce(buyQuoteUpdater.updateBuyQuoteAsync.bind(buyQuoteUpdater), 200, {
     trailing: true,
-});
+}) as typeof buyQuoteUpdater.updateBuyQuoteAsync;
```

Comment:
without this, `debouncedUpdateBuyQuoteAsync` was `any` 😱

Reply:
🙀